### PR TITLE
Add jwt or native guard

### PR DESCRIPTION
**Pull request by danielailie**

### CODE REVIEW

File: packages/common/src/guards/jwt.or.native.auth.guard.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import {
+  Injectable,
+  CanActivate,
+  ExecutionContext,
+  Inject,
+} from "@nestjs/common";
+import { CachingService } from "src/common/caching/caching.service";
```
Comment (gabrielmatei) [inline packages/common/src/guards/jwt.or.native.auth.guard.ts:7]: use relative paths for imports

File: packages/common/src/guards/jwt.or.native.auth.guard.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import { ErdnestConfigService } from "src/common/config/erdnest.config.service";
+import { ERDNEST_CONFIG_SERVICE } from "src/utils/erdnest.constants";
+import { JwtAuthenticateGuard } from "./jwt.authenticate.guard";
+import { NativeAuthGuard } from "./native.auth.guard";
+
+@Injectable()
+export class JwtOrNativeAuthGuard implements CanActivate {
```
Comment (gabrielmatei) [inline packages/common/src/guards/jwt.or.native.auth.guard.ts:14]: export file in `index.ts`

**Review by gabrielmatei — CHANGES_REQUESTED** [review]

File: packages/common/src/guards/jwt.or.native.auth.guard.ts (hunk 1 above)
Comment (danielailie) [reply packages/common/src/guards/jwt.or.native.auth.guard.ts]: done

File: packages/common/src/guards/jwt.or.native.auth.guard.ts (hunk 2 above)
Comment (danielailie) [reply packages/common/src/guards/jwt.or.native.auth.guard.ts:9]: done

**Review by gabrielmatei — APPROVED** [review]

**Review by johnykes — APPROVED** [review]

**Review by gabrielmatei — APPROVED** [review]

**Review by tanghel — APPROVED** [review]

**Pull request merged**